.dev/systemimage/climate_machine_image.jl

```diff
@@ -6,33 +6,47 @@
 #
 # Called with a single argument the system image will be placed in the path
 # specified by the argument (relative to the callers path)
+#
+# Called with a specified systemimg path and `true`, the system image will
+# compile the climate machine package module (useful for CI)
+
 sysimage_path =
-    isempty(ARGS) ? joinpath(@__DIR__, "ClimateMachine.so") : ARGS[1]
+    len(ARGS) < 1 ? joinpath(@__DIR__, "ClimateMachine.so") : abspath(ARGS[1])
```

Comment:
Still writing Python?
```suggestion
    isempty(ARGS) ? joinpath(@__DIR__, "ClimateMachine.so") : abspath(ARGS[1])
```

Reply:
yes :disappointed: